opengate/actors/coincidences.py

```diff
@@ -1,6 +1,67 @@
 import awkward as ak
 from tqdm import tqdm
 from ..exception import fatal
+from .base import ActorBase
+
+
+class Coincidences(ActorBase):
```

Comment:
Add the class to __init__ file: https://github.com/OpenGATE/opengate/blob/master/opengate/actors/__init__.py

Reply:
Done